### ISSUE TITLE
Writing dates as parquet datetime types

### ISSUE THREAD

**Issue by ptyrlik1** [Cinchoo/ChoETL#288]
I have a program that writes a list of objects of a specific type to parquet. The issue is when it is writing date properties to the parquet file they are saved as strings rather than a datetime type. 

This is how I have my parser configured 

```
using (var parser = new ChoParquetWriter(outSteam)
.Configure(c => c.Culture = CultureInfo.InvariantCulture)
.Configure(c => c.TypeConverterFormatSpec = new ChoTypeConverterFormatSpec { DateTimeFormat = "o" })

```

This is the definition of the property in the class

`public DateTime? date_reported { get; set; }`

And this is a example of what the date looks in the database I am reading from 

2023-01-09 00:00:00.000

And this is how it is stores in the object 
![image](https://github.com/Cinchoo/ChoETL/assets/121582045/c9cfaaf7-025f-4856-a4e2-156b21b71245)

**Comment by ptyrlik1** [comment]
Is there a datetime like type that it does support such as datetimeoffset?

**Comment by Cinchoo** [comment]
yes, there is way to use datetimeoffset. let me add it. Will update.

**Comment by ptyrlik1** [comment]
Did you push this update and if so how is it used?

**Comment by Cinchoo** [comment]
Yes, here is how you can control the output

```
                using (var w = new ChoParquetWriter(filePath)
                    .Configure(c => c.TreatDateTimeAsDateTimeOffset = true)
                    )
                {
                    w.Write(recs);
                }
```
